README.md

```diff
@@ -1,30 +1,29 @@
 # Raiden Explorer
 
-Requires NodeJS and npm.
+[The Raiden Explorer](https://explorer.raiden.network/home) displays information about the status of the [Raiden Network](https://raiden.network/).
 
-## Mock endpoint
-Serv test data by running mock endpoint on `localhost:4567`:
-```bash
-cd mock
-node serv.js
+## Running
+
+The Explorer can be run by using `docker-compose`:
+```sh
+docker-compose build
+docker-compose up -d
 ```
 
-## Dev run
-Install angular globally and run the ng dev server:
+It's important to update the [frontend config files](https://github.com/raiden-network/explorer/tree/master/frontend/src/assets/config) because otherwise the official Explorer backend is used.
+
+## Frontend
+
+The frontend is built using [AngularJS](https://angularjs.org/) and required NodeJS. It can be found in the `frontend` directory.
```

Comment:
Actually it is [Angular](https://angular.io/). `AngularJS` is the legacy framework based on Angular v1.x